lib/provider/images/image_provider.dart

```diff
@@ -1,20 +1,14 @@
-import 'package:hooks_riverpod/hooks_riverpod.dart';
 import 'package:firebase_storage/firebase_storage.dart';
 
-final mapImagesProvider = FutureProvider<List<String>?>(
-  (ref) async => await getMapImages(),
-);
-
-Future<List<String>?> getMapImages() async {
+Future<List<String>?> getMapImages(String storagePath) async {
```

Comment:
Zmieniłbym nazwę tej funkcji z getMapImages na getImages ze względu na to że nie jest już tylko używana do zdjęć mapy.